### DISCUSSION TITLE
Welcome to SrvSurvey Discussions!

### DISCUSSION THREAD

**Discussion by njthomson** [njthomson/SrvSurvey#138] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wisekeep** [comment]
I use the Microsoft Store version, is there an estimated date for when an update will be released there?

Thank you for this app, it saved me many hours of gameplay, thank you.

**Comment by Taz-7** [comment]
Feature update request:

In the Search/Boxel/Config, extend the Mass code to e/f/g/h.

Thanks much, made exploring SO much easier!

Taz.

**Reply by njthomson** [reply]
Hi @Taz-7 - did you mean this little drop-down?

<img width="522" height="225" alt="image" src="https://github.com/user-attachments/assets/5c8be94a-c1d2-4291-922e-95606ec314bb" />

It is used to specify how far "down" to go, and it should include all mass codes smaller than the starting system shown just above it. 

As including each smaller mass code means you have 8x sub-boxels to search - the point of this drop down is for you to limit things "from the current level" and go "down" only as many mass codes as you can stomach. As shown, if I start surveying that G boxel and include down to D boxels ... I would need to survey 585 child boxels *(with who knows how many systems within them)*

Also: as you are there, you can also the starting system using the `...` button to navigate "up" to the parent boxel, "down" into child boxels, and laterally between the 7 siblings:

<img width="278" height="401" alt="image" src="https://github.com/user-attachments/assets/09a37d30-52fb-4e8d-acb2-d80541a0731c" />

**Comment by Taz-7** [comment]
Yes... but hmmm, my drop down only shows a-d, that's why I asked.   I'll try the currently available version and see if I just had an old version. Thanks!Taz.From: njthomson ***@***.***>Sent: Thursday, June 11, 2026 1:00:52 p.m.To: njthomson/SrvSurvey ***@***.***>Cc: Taz-7 ***@***.***>; Mention ***@***.***>Subject: Re: [njthomson/SrvSurvey] Welcome to SrvSurvey Discussions! (Discussion #138)Hi @Taz-7 - did you mean this little drop-down?image.png (view on web)
It is used to specify how far "down" to go, and it should include all mass codes smaller than the starting system shown just above it.As including each smaller mass code means you have 8x sub-boxels to search - the point of this drop down is for you to limit things "from the current level" and go "down" only as many mass codes as you can stomach. As shown, if I start surveying that G boxel and include down to D boxels ... I would need to survey 585 sub-boxels (with who knows how many systems within them)You can adjust the starting system using the ... button to navigate "up" to the parent boxel, "down" into child boxels, and laterally between the 7 siblings:image.png (view on web)
